C_routing.md

```diff
@@ -778,113 +778,50 @@ end
 
 Channels are a very exciting, real-time component of the Phoenix framework. Channels handle incoming and outgoing messages broadcast over a socket for a given topic. Channel routes, then, need to match requests by socket and topic in order to dispatch to the correct channel. (For a more detailed description of channels and their behavior, please see the [Channel Guide](http://www.phoenixframework.org/docs/channels).)
 
-We define sockets in the router with a block, a path to the socket's mount point, and (optionally) the name of our application to fully qualify our channel module name. This should look familiar after seeing our scope definitions above.
-
-Here's what that looks like in our router file:
+We mount socket handlers in our endpoint, who handle authentication callbacks and channel routes.
```

Comment:
How about this?

We mount socket handlers in our endpoint. Socket handlers take care of authentication callbacks and channel routes.